sandbox_store.go

```diff
@@ -240,6 +246,10 @@ func (c *controller) sandboxCleanup(activeSandboxes map[string]interface{}) {
 			heap.Push(&sb.endpoints, ep)
 		}
 
+		for _, dns := range sbs.ExtDNS {
```

Comment:
Isn't it better to recreate the slice in the new sandbox ? holding on to the reference of sbs.extDNS would prevent  sbState from being GC'ed.